package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "react-streamfield",
+  "version": "0.6.10",
+  "author": "NoriPyt <contact@noripyt.com> (https://noripyt.com)",
+  "description": "Powerful field for inserting multiple blocks with nesting.",
+  "keywords": [
+    "react",
+    "react-component",
+    "field"
+  ],
+  "homepage": "https://github.com/noripyt/react-streamfield",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/noripyt/react-streamfield"
+  },
+  "bugs": {
+    "url": "https://github.com/noripyt/react-streamfield/issues"
+  },
+  "license": "BSD",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "main": "src/index.js",
+  "devDependencies": {
+    "@babel/cli": "7.1.5",
+    "@babel/core": "7.1.6",
+    "@babel/plugin-proposal-class-properties": "7.1.0",
+    "@babel/plugin-proposal-decorators": "7.1.6",
+    "@babel/plugin-proposal-object-rest-spread": "7.0.0",
+    "@babel/preset-env": "7.1.6",
+    "@babel/preset-react": "7.0.0",
+    "babel-core": "7.0.0-bridge.0",
+    "babel-loader": "8.0.4",
+    "babel-plugin-transform-react-remove-prop-types": "0.4.21",
+    "classnames": "2.2.6",
+    "css-loader": "1.0.1",
+    "mini-css-extract-plugin": "0.4.5",
+    "node-sass": "4.10.0",
+    "prop-types": "15.6.2",
+    "react": "16.6.3",
+    "react-animate-height": "2.0.7",
+    "react-beautiful-dnd": "10.0.2",
+    "react-dom": "16.6.3",
+    "react-redux": "5.1.1",
+    "redux": "4.0.1",
+    "redux-thunk": "2.3.0",
```

Comment:
It would also be worth moving `redux`-related dependencies to `peerDependencies`, since Wagtail uses them already and we don't want to have two versions of Redux for not much value.